controller/config.go

```diff
@@ -456,7 +456,7 @@ const (
 
 	// DefaultOpenTelemetryTailSamplingThreshold is the default value for the
 	// tail sampling threshold for open telemetry.
-	DefaultOpenTelemetryTailSamplingThreshold = 1 * time.Microsecond
+	DefaultOpenTelemetryTailSamplingThreshold = 1 * time.Millisecond
```

Comment:
Drive-by 2c: it's still way low. Something like 200ms maybe?

Reply:
For now, we want everything to be sampled hence the low value, in production cases you're probably right, that this should be higher. This is to prevent us from OOM'ing when we've got so many traces locally, and trying to batch them to the server.